runtime/src/lib.rs

```diff
@@ -397,10 +416,11 @@ macro_rules! construct_vln_runtime {
                     Sudo: pallet_sudo::{Module, Call, Storage, Config<T>, Event<T>},
 
                     // vln dependencies
-                    Tokens: orml_tokens::{Config<T>, Event<T>, Module, Storage},
-                    Proxy: pallet_proxy::{Call, Event<T>, Module, Storage},
+                    TokensGeneralInstance: orml_tokens::<Instance1>::{Config<T>, Event<T>, Module, Storage},
+                    TokensCollateralInstance: orml_tokens::<Instance2>::{Config<T>, Event<T>, Module, Storage},
```

Comment:
Maybe can we just call it `Collateral`? 🙂 just a nitpick to be able to query the state with the VLN API doing something like `GET /collateral/total-issuance/ausd` instead of `/tokens-collateral-instance/total-issuance/ausd`

Reply:
In the same sense we could keep calling the `TokensGeneralInstance` just `Tokens`, what do you think?